vocabularies/voc4cat/0000201.ttl

```diff
@@ -0,0 +1,14 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0000201>
+    a skos:Concept ;
+    dcterms:identifier "voc4cat_0000201"^^xsd:token ;
+    skos:altLabel "cross-coupling"@en ;
+    skos:broader <https://w3id.org/nfdi4cat/voc4cat_0000181> ;
+    skos:definition "A cross-coupling reaction is a catalytic, metal-mediated reaction that forms carbon-carbon or carbon-heteroatom bonds by joining two different molecular fragments, typically through the activation of electrophilic and nucleophilic partners."@en ;
+    skos:historyNote "0000-0002-6242-2167 Nikolaos G. Moustakas created the resource"@en ;
+    skos:inScheme <https://w3id.org/nfdi4cat/voc4cat> ;
+    skos:prefLabel "cross-coupling reaction (or cross-coupling)"@en ;
```

Comment:
remove "(or cross-coupling)" from prefLabel. Nothing else to do; it is already an altLabel.